web/client/components/data/featuregrid/FeatureEditorFallback.jsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useMemo } from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import { getDockSize } from '../../../selectors/featuregrid';
```

Comment:
This selector is including a lot of dependencies for this specific case I would suggest to write here directly the selector and remove this import

Reply:
Done